responses/lv/HassLightSet.yaml

```diff
@@ -0,0 +1,8 @@
+language: lv
+responses:
+  intents:
+    HassLightSet:
+      brightness: '{{ slots.name }} brightness set to {{ slots.brightness }}'
+      brightness_area: Brightness in {{ slots.area }} set to {{ slots.brightness }}
+      color: '{{ slots.name }} color set to {{ slots.color }}'
+      color_area: Color in {{ slots.area }} set to {{ slots.color }}
```

Comment:
So, what standard do we use? Do we use single quotes, double quotes, or no quotes at all? 😄

Reply:
Anyone know how to force PyYAML to always export with double quotes? I'd prefer that, but haven't found a way to do it let :/